docs/getting-started/docker-playgrounds/flink_cluster_playground.md

```diff
@@ -0,0 +1,774 @@
+
+This playground consists of a long living
+[Flink Session Cluster]({{ site.baseurl }}/concepts/glossary.html#flink-session-cluster) and a Kafka
+Cluster.
+
+A Flink Cluster always consists of a 
+[Flink Master]({{ site.baseurl }}/concepts/glossary.html#flink-master) and one or more 
+[Flink TaskManagers]({{ site.baseurl }}/concepts/glossary.html#flink-taskmanager). The Flink Master 
+is responsible for handling [Job]({{ site.baseurl }}/concepts/glossary.html#flink-job) submissions, 
+the supervision of Jobs as well as resource management. The Flink TaskManagers are the worker 
+processes and are responsible for the execution of the actual 
+[Tasks]({{ site.baseurl }}/concepts/glossary.html#task) which make up a Flink Job. In this 
+playground you will start with a single TaskManager, but scale out to more TaskManagers later. 
+Additionally, this playground comes with a dedicated *client* container, which we use to submit the 
+Flink Job initially and to perform various operational tasks later on. The *client* container is not
+needed by the Flink Cluster itself but only included for ease of use.
+
+The Kafka Cluster consists of a Zookeeper server and a Kafka Broker.
+
+<img src="{{ site.baseurl }}/fig/flink-docker-playground.svg" alt="Flink Docker Playground"
+class="offset" width="80%" />
+
+When the playground is started a Flink Job called *Flink Event Count* will be submitted to the 
+Flink Master. Additionally, two Kafka Topics *input* and *output* are created.
+
+<img src="{{ site.baseurl }}/fig/click-event-count-example.svg" alt="Click Event Count Example"
+class="offset" width="80%" />
+
+The Job consumes `ClickEvent`s from the *input* topic, each with a `timestamp` and a `page`. The 
+events are then keyed by `page` and counted in 15 second
+[windows]({{ site.baseurl }}/dev/stream/operators/windows.html). The results are written to the 
+*output* topic. 
+
+There are six different pages and we generate 1000 click events per page and 15 seconds. Hence, the 
+output of the Flink job should show 1000 views per page and window.
+
+{% top %}
+
+## Starting the Playground
+
+{% if site.version contains "SNAPSHOT" %}
+<p style="border-radius: 5px; padding: 5px" class="bg-danger">
+  <b>Note</b>: The Apache Flink Docker images used for this playground are only available for
+  released versions of Apache Flink. Since you are currently looking at the latest SNAPSHOT
+  version of the documentation the branch referenced below will not exist. You can either change it 
+  manually or switch to the released version of the documentation via the release picker.
+</p>
+{% endif %}
+
+The playground environment is set up in just a few steps. We will walk you through the necessary 
+commands and show how to validate that everything is running correctly.
+
+We assume that you have that you have [docker](https://docs.docker.com/) (1.12+) and
+[docker-compose](https://docs.docker.com/compose/) (2.1+) installed on your machine.
```

Comment:
Sorry late to reply, I just noticed that here should be a docker-compose file version instead of a docker-compose version. Just a minor advice.